deploy-disconnected-registry/deploy.sh

```diff
@@ -241,14 +241,24 @@ function deploy_registry() {
 
 if [[ ${1} == 'hub' ]]; then
     if ! ./verify.sh 'hub'; then
-        deploy_registry 'hub'
-        trust_internal_registry 'hub'
-        check_resource "deployment" "${REGISTRY}" "Available" "${REGISTRY}" "${KUBECONFIG_HUB}"
-        check_mcp 'hub'
-        render_file manifests/machine-config-certs-master.yaml 'hub'
-        render_file manifests/machine-config-certs-worker.yaml 'hub'
-        check_resource "mcp" "master" "Updated" "default" "${KUBECONFIG_HUB}"
-        check_resource "deployment" "${REGISTRY}" "Available" "${REGISTRY}" "${KUBECONFIG_HUB}"
+        # Check if private registry 
+        if [[ ${CUSTOM_REGISTRY} == "false" ]]; then
+            deploy_registry 'hub'
+            trust_internal_registry 'hub'
+            render_file manifests/machine-config-certs-master.yaml 'hub'
```

Comment:
please, don't change the order...there was a corner case that Juan tried to solve with this order (waiting mcp and resources before rendering the file